loris/resolver.py

```diff
@@ -61,13 +65,40 @@ def resolve(self, ident):
             ident (str):
                 The identifier for the image.
         Returns:
-            (str, str): (fp, format)
+            ImageInfo: Partially constructed ImageInfo object 
```

Comment:
Ahha, thank you!  Back to green.

After a bit of discussion with our new Enterprise Software Architect, we decided to replace the current, custom cryptography code with a higher level spec -- JSON Web Tokens.  I haven't done this yet, but should get it finished today.